src/common/database.c

```diff
@@ -1226,6 +1226,15 @@ static int _upgrade_data_schema_step(dt_database_t *db, int version)
 
     new_version = 2;
   }
+  else if(version == 2)
+  {
+    TRY_EXEC("ALTER TABLE data.tags RENAME COLUMN description TO synonyms;",
```

Comment:
I'm surprised, is that really working now? Last time I checked the RENAME COLUMN was not implemented in SQLite. Is that fixed now? In which SQLite version?